client/packages/common/src/intl/locales/en/patients.json

```diff
@@ -1,7 +1,12 @@
 {
   "button.new-patient": "New Patient",
   "filename.patients": "patients",
+  "label.date-of-birth": "DOB (Age)",
+  "label.details": "Details",
+  "label.gender": "Gender",
```

Comment:
This is requested to be "Sex". Do we usually call it gender? do we need to make it configurable?

Reply:
we've used `Gender` previously and to me it's a gentler term to use for this. Given that these changes are specific for a client, they get to call the shots though.. I guess.